src/main/java/com/flagsmith/FlagsmithConfig.java

```diff
@@ -37,6 +40,9 @@ protected FlagsmithConfig(Builder builder) {
         if (builder.sslSocketFactory != null && builder.trustManager != null) {
             httpBuilder = httpBuilder.sslSocketFactory(builder.sslSocketFactory, builder.trustManager);
         }
+        for (Interceptor interceptor: builder.interceptor) {
```

Comment:
Not sure what the HTTP builder interface looks like but could this just be `httpBuilder.setInterceptors(builder.interceptors)` ?

Reply:
The interface does not offer such option unfortunately.